src/Documentation/tutorials_and_samples/tutorial_1_helloworld.md

```diff
@@ -0,0 +1,377 @@
+---
+layout: page
+title: Tutorial 1 Hello World
+---
+
+# Tutorial 1: Hello World
+
+This process recreates the same Hello World sample application available [here](https://github.com/dotnet/orleans/tree/master/Samples/2.0/HelloWorld).
+
+## Overview of the parts
+
+This application consists of a solution that contains four projects: SiloHost, OrleansClient, HelloWorld.Interfaces, and HelloWorld.Grains.
+We will create the SiloHost and OrleansClient projects as .NET Core App files and we will create the HelloWorld.Interfaces and HelloWorld.Grains projects as .NET Standard Libraries.
+Orleans comes into the picture when we add the NuGet packages to each of these four projects and replace the default code with the Orleans code provided in this tutorial.
+Finally, we will configure the dependency references among the projects before building and running the solution.
+
+## How the parts work together
+
+SiloHost is started first.
+Then, OrleansClient is started. OrleansClient tells the system to create a grain based on the grain interface provided.
+The grain is then stored in SiloHost. OrleansClient sends a greeting to the grain.
```

Comment:
```suggestion
the OrleansClient call to the grain reference would then activate the grain in the silo host. And the activated grain would then process SayHello() call
```

grains are activated, not stored.

Reply:
Thanks! I will change any other wording like that, as I come across it.